staging/src/k8s.io/legacy-cloud-providers/azure/azure.go

```diff
@@ -345,7 +345,7 @@ func (az *Cloud) InitializeCloudFromConfig(config *Config, fromSecret bool) erro
 	}
 
 	// Conditionally configure resource request backoff
-	resourceRequestBackoff := wait.Backoff{
+	ResourceRequestBackoff := wait.Backoff{
```

Comment:
for the local variable, we usually start with lower case

Reply:
good catch, fixed.